AppLibraryEnabler.xm

```diff
@@ -136,22 +132,6 @@ typedef struct SBHIconGridSize {
 	contentContainerView.frame = selfFrame;
 	searchResultsContainerView.frame = selfFrame;
 }
-- (void)_layoutSearchViews {
```

Comment:
This part will be needed for application list (only height and top offset part) - without it overlay in this view is cut out on top and bottom.